google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/spi/v1/Publisher.java

```diff
@@ -150,14 +151,24 @@
     static final long DEFAULT_MAX_BUNDLE_BYTES = 1000L; // 1 kB
     static final Duration DEFAULT_MAX_BUNDLE_DURATION = new Duration(1); // 1ms
     static final Duration DEFAULT_REQUEST_TIMEOUT = new Duration(10 * 1000); // 10 seconds
-    static final Duration MIN_SEND_BUNDLE_DURATION = new Duration(10 * 1000); // 10 seconds
-    static final Duration MIN_REQUEST_TIMEOUT = new Duration(10); // 10 milliseconds
+    static final Duration MIN_TOTAL_TIMEOUT = new Duration(10 * 1000); // 10 seconds
+    static final Duration MIN_RPC_TIMEOUT = new Duration(10); // 10 milliseconds
     static final BundlingSettings DEFAULT_BUNDLING_SETTINGS =
         BundlingSettings.newBuilder()
             .setDelayThreshold(DEFAULT_MAX_BUNDLE_DURATION)
             .setRequestByteThreshold(DEFAULT_MAX_BUNDLE_BYTES)
             .setElementCountThreshold(DEFAULT_MAX_BUNDLE_MESSAGES)
             .build();
+    static final RetrySettings DEFAULT_RETRY_SETTINGS =
+        RetrySettings.newBuilder()
+            .setTotalTimeout(MIN_TOTAL_TIMEOUT)
```

Comment:
We should distinguish the concepts of minimum and default - make a new constant `DEFAULT_TOTAL_TIMEOUT` and use that here.